static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,193 @@
+import React, {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Duration from 'sentry/components/duration';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import UserBadge from 'sentry/components/idBadge/userBadge';
+import Link from 'sentry/components/links/link';
+import Placeholder from 'sentry/components/placeholder';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCalendar} from 'sentry/icons';
+import space from 'sentry/styles/space';
+import {NewQuery, PageFilters} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {generateEventSlug} from 'sentry/utils/discover/urls';
+import getUrlPathname from 'sentry/utils/getUrlPathname';
+import theme from 'sentry/utils/theme';
+import {useLocation} from 'sentry/utils/useLocation';
+import useMedia from 'sentry/utils/useMedia';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+import indexReplayEntriesById from './utils/indexReplayEntriesById';
+import {Replay} from './types';
+
+type Props = {
+  replayList: Replay[];
+  selection: PageFilters;
+};
+
+export type ReplayDurationAndErrors = {
```

Comment:
Does this need to be exported?

Reply:
we are using it on the `indexReplayEntriesById` file to make the new array.